pkg/container/vector/tools.go

```diff
@@ -35,6 +36,19 @@ func MustTCols[T types.FixedSizeT](v *Vector) []T {
 	panic("unexpected parameter types were received")
 }
 
+func MustTColsInteger[T constraints.Integer](v *Vector) []T {
```

Comment:
Why do you need this?   MustTCols does not work?

Reply:
yes, the MustTCols does not work in this case.

Reply:
Why?

Reply:
I find it ok today, my fault, I will use the MustTCols.